tslearn/neighbors.py

```diff
@@ -286,6 +296,9 @@ def predict(self, X):
                 X_ = cdist_dtw(X, self._ts_fit, **metric_params)
             elif self._ts_metric == "softdtw":
                 X_ = cdist_soft_dtw(X, self._ts_fit, **metric_params)
+            else:
```

Comment:
Will this code ever be called? As it is enclosed within `if self.metric in self.variable_length_metrics:`. It is a good check though for in case a metric ever gets added to that list but the corresponding code isn't written here

Reply:
The code was there before the PR, but since it is related I will mention this here.

I think it would be cleaner to have `variable_length_metrics` defined in `metrics.py`, and to make it a constant, like `sklearn.metrics.SCORERS`. Constants are written with all caps, so it should be `VARIABLE_LENGTH_METRICS`.